apps/projects/app/components/App/App.js

```diff
@@ -372,15 +374,17 @@ class App extends React.PureComponent {
       'onReviewWork',
       web3.toHex(issue.repoId),
       issue.number,
-      issue.assignee,
-      state.accepted
+      issue.workSubmissions[issue.workSubmissions.length - 1],
+      state.accepted,
+      issue.workSubmissions[issue.workSubmissions.length - 1].submissionIPFSHash
     )
     this.closePanel()
     this.props.app.reviewSubmission(
       web3.toHex(issue.repoId),
       issue.number,
-      issue.assignee,
-      state.accepted
+      issue.workSubmissions.length - 1,
```

Comment:
Is this what you were talking about with only the ability to review the most recent submission

Reply:
no, I added this line to conform to the current workflow. This should eventually probably be replaced by an array index, unless it just makes sense to only accept the most recent submission anyhow.